.github/workflows/publish-release.yml

```diff
@@ -25,3 +25,13 @@ jobs:
       - uses: MetaMask/action-publish-release@v0.1.0
         env:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+      - name: Install and Build and Deploy
+        shell: bash
+        run: |
+          yarn setup
+          git worktree add public gh-pages
+          yarn build
```

Comment:
I think this would continue on if the build failed, because Bash will not stop upon failure by default.

Probably worth migrating this to an external Bash script or something?